deployment/postgres-init-db/sql/tables/merlin/constraints/constraint_run.sql

```diff
@@ -10,7 +12,7 @@ create table merlin.constraint_run (
   requested_at timestamptz not null default now(),
 
   constraint constraint_run_key
-    primary key (constraint_id, constraint_revision, simulation_dataset_id),
+    primary key (constraint_invocation_id, simulation_dataset_id),
```

Comment:
Blocking: This pkey is not sufficient. If the user decides to check constraints twice, updating the definition of a constraint between checks, they will get a pkey conflict when the action attempts to cache the output of the second defintion.